APMrover2/config.h

```diff
@@ -165,70 +165,70 @@
 // GROUND_START_DELAY
 //
 #ifndef GROUND_START_DELAY
-# define GROUND_START_DELAY		0
+  #define GROUND_START_DELAY      0
 #endif
 
 //////////////////////////////////////////////////////////////////////////////
 // MOUNT (ANTENNA OR CAMERA)
 //
 #ifndef MOUNT
-# define MOUNT		ENABLED
+  #define MOUNT                   ENABLED
 #endif
 
 //////////////////////////////////////////////////////////////////////////////
 // CAMERA control
 //
 #ifndef CAMERA
-# define CAMERA ENABLED
+  #define CAMERA ENABLED
 #endif
 
 //////////////////////////////////////////////////////////////////////////////
 // AIRSPEED_CRUISE
 //
 #ifndef SPEED_CRUISE
-# define SPEED_CRUISE		5 // in m/s
+  #define SPEED_CRUISE   5  // in m/s
 #endif
 
 #ifndef TURN_GAIN
-# define TURN_GAIN		5
+  #define TURN_GAIN      5
 #endif
 
 //////////////////////////////////////////////////////////////////////////////
 // Servo Mapping
 //
 #ifndef THROTTLE_MIN
-# define THROTTLE_MIN			0 // percent
+  #define THROTTLE_MIN        0  // percent
 #endif
 #ifndef THROTTLE_CRUISE
-# define THROTTLE_CRUISE		45
+  #define THROTTLE_CRUISE     45
 #endif
 #ifndef THROTTLE_MAX
-# define THROTTLE_MAX			100
+  #define THROTTLE_MAX        100
 #endif
 
 //////////////////////////////////////////////////////////////////////////////
 // Attitude control gains
 //
 #ifndef SERVO_STEER_P
-# define SERVO_STEER_P         0.4
+  #define SERVO_STEER_P         0.4
 #endif
 #ifndef SERVO_STEER_I
-# define SERVO_STEER_I         0.0
+  #define SERVO_STEER_I         0.0
 #endif
 #ifndef SERVO_STEER_D
-# define SERVO_STEER_D         0.0
+  #define SERVO_STEER_D         0.0
 #endif
 #ifndef SERVO_STEER_INT_MAX
-# define SERVO_STEER_INT_MAX   5
+  #define SERVO_STEER_INT_MAX   5
 #endif
-#define SERVO_STEER_INT_MAX_CENTIDEGREE SERVO_STEER_INT_MAX*100
+#define SERVO_STEER_INT_MAX_CENTIDEGREE (SERVO_STEER_INT_MAX * 100)
 
 
 //////////////////////////////////////////////////////////////////////////////
 // Dataflash logging control
 //
 #ifndef LOGGING_ENABLED
-# define LOGGING_ENABLED		ENABLED
+  #define LOGGING_ENABLED     ENABLED
```

Comment:
Actually I like that the define is indented (and I don't care if the # is at the beginning of the line or after spaces). I was referring to the spaces after the define name, this file isn't consistent at all, some places have a huge number of spaces there, others have one space only.

Like I said, it would be nice to make the whole file look better, but I'm not requesting it as it isn't worst than before.